deployment/systemd/manager-web-server.service

```diff
@@ -3,6 +3,7 @@ Description=Radio app manager web server
 
 [Service]
 EnvironmentFile=/opt/radiodan/rde/deployment/systemd/ports.env
+EnvironmentFile=-/opt/radiodan/rde/services/setup/tmp/apps
```

Comment:
The `-` indicates that is an optional file, so `systemd` will not crash if it's not found.

https://www.freedesktop.org/software/systemd/man/systemd.exec.html#EnvironmentFile=

Reply:
Can you add this as a comment to the service file? It's definite one of those things future-me will forget.